src/cicd/pipelines.ts

```diff
@@ -156,6 +173,40 @@ export class CICDPipelineStack extends Stack {
     return this;
   }
 
+  addWave(props: AddApplicationWaveProps) {
+    /*
+    Add multiple application stages in parallel to the CICD pipeline. This stage deploys your application infrastructure.
+      Parameters
+    ----------
+    stageId: str
+    Identifier of the wave
+    stages: Stage[]
+    Application stage instance
+    manualApprovals: Optional[bool]
+    Configure manual approvals. False by default
+      Returns
+    -------
+    pipeline : CICDPipelineStack
+    CICDPipelineStack
+    */
+    if (this.pipeline === undefined) {
+      throw new Error('`.buildPipeline()` needs to be called first before adding application stages to the pipeline.');
+    }
+    var manualApprovals = props.manualApprovals ?? false; // || this._config.get_env_config(stage_id).get('manual_approvals');
```

Comment:
In Typescript, `const` is generally preferable to `var`. The `var` keyword should only be used when we expect the value to change, like for a counter.